inbc-program/tests/unit/test_parser.py

```diff
@@ -85,11 +85,12 @@ def test_remove_manifest(self) -> None:
                    '</configtype></config></manifest>'
         self.assertEqual(remove.func(remove), expected)
 
-    def test_raise_not_supported_restart_manifest(self) -> None:
-        restart = self.arg_parser.parse_args(['restart'])
-        with self.assertRaisesRegex(InbcException, 'Restart command is not supported.'):
-            restart.func(restart)
-
+    def test_create_restart_manifest(self):
+        s = self.arg_parser.parse_args(['restart'])
+        
+        expected = '<?xml version="1.0" encoding="utf-8"?><manifest><type>cmd</type><cmd>restart</cmd></manifest>'
```

Comment:
**<h2>$\color{#FFA500}Improvements: Code Refactoring$</h2>**
**Suggestion:** Remove the redundant '+' operator in the string concatenation for the XML manifest as it is unnecessary in Python when using parentheses for implicit line continuation.
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
        expected = ('<?xml version="1.0" encoding="utf-8"?>'
                    '<manifest>'
                    '<type>cmd</type>'
                    '<cmd>restart</cmd>'
                    '</manifest>')
```